src/brackets.js

```diff
@@ -371,6 +371,42 @@ define(function (require, exports, module) {
             });
         });
 
+        // Check If Adobe XD Application is installed on User Machine
```

Comment:
Should we really pollute `brackets.js` with this piece of code? Can we think about moving this somewhere else?
@swmitra for thoughts?

Reply:
Perhaps we can move all this to an extension without changing any code files?